docs/requisitos/elicitacao/brainstorming.md

```diff
@@ -57,24 +61,31 @@ A Etapa de transcrição ocorreu tanto durante a **Etapa 1** quanto na **Etapa 2
 
 De forma geral, foi possível notar um afunilamento dos tópicos da tabela anterior com os épicos do [Backlog](../../modelagem/backlog_geral) montado bem como dos servições analisados na arquitetura.
 
-### Etapa 4 - Prototipação
+### 3.4. Etapa 4 - Prototipação
 
 Para alinhamento da equipe com a aplicação elaborada após as etapas mais abstratas, foi realizada uma prototipação conceitual, contando com apoio da representação visual no quadro branco para a elaboração de alguns fluxos da aplicação, como o fluxo de login, cadastro e traduções.
 
-#### Protótipo - Login/Cadastro do Autor
+#### 3.4.1. Protótipo - Login/Cadastro do Autor
```

Comment:
It's not the login prototype, is the flux to send a new translation request,it's written login in the beginning just to represent the action